src/components/Drawer.vue

```diff
@@ -168,7 +168,7 @@ export default class Drawer extends Vue {
 
   created() {
     const searchMenuItem = {
-      icon: 'mdi-card-search-outline',
+      icon: 'mdi-text-box-search-outline',
```

Comment:
`cloud-search-outline` or `text-search` better.

Reply:
Resolved in 24a631a.